docs/pages/connect-your-client/introduction.mdx

```diff
@@ -20,6 +20,9 @@ your Teleport cluster:
 <Tabs>
 <TabItem label="Local user">
 
+After entering your Teleport variables above you can copy and paste the below command.  You should see similar output
```

Comment:
Something about the wording here doesn't feel right. Yes, you can copy and paste the command, but I think we should discuss what it does or why you would want to.

Also note that we use a single space between sentences, not two.

Reply:
Also, you can input page Vars anywhere they are used, it doesn't have to be at the VarList component.

Reply:
> I think we should discuss what it does or why you would want to.

That's what the paragraph above the Tabs says, which I think renders this entire addition redundant.